EIPS/eip-7492.md

```diff
@@ -0,0 +1,333 @@
+---
+eip: 7492
+title: ERC-20Spendable
```

Comment:
This title is not descriptive. Please use a title like "ERC-20 <whatever> Extension".